devtools-html/debugger.html
# [SourcesTree] Style debuggee URL domain

Associated Issue: #4259

### Summary of Changes

* added new svg style for domain icon
* wrote code to highlight debuggee domain in light green

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

- start debugger
- start debugging a web page
- observe the main domain being light green

### Screenshots/Videos (OPTIONAL)
<img width="240" alt="1" src="https://user-images.githubusercontent.com/31850821/31856513-c0abf0e0-b690-11e7-8e1c-618bb86e595c.png">
